actor/src/broker.rs

```diff
@@ -131,7 +131,7 @@ impl Broker {
                 }
             })
             .collect::<FuturesUnordered<_>>()
-            .collect::<Vec<_>>()
+            .collect::<smallvec::SmallVec<[_; 8]>>()
```

Comment:
is this related to `usize`?

Reply:
Actually, CPU cores. Sadly there can be no way of getting that information at compile time, so 8 seemed like a good guess.

Reply:
how is # of CPU cores related to stack size of the vec?
would it make sense to extract into a type or a const?